### PR TITLE
Infra 27534 add nlb support

### PR DESCRIPTION
**Pull request by barrymars**

### CODE REVIEW

File: charts/standard-application-stack/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v3.49.0] - 2022-10-04
+### Added
+- Added support for Network Load Balancers off of LoadBalancer services
```
Comment (nabadger) [inline charts/standard-application-stack/CHANGELOG.md:12]: Comments seems a bit strange "off of loadbalancer services" ?

File: charts/standard-application-stack/README.md [hunk 2]
```diff
@@ -186,6 +186,15 @@ A generic chart to support most common application requirements
 | minReadySeconds | int | `10` | Minimum number of seconds before deployments are ready |
 | nameOverride | string | `""` | String to fully override mintel_common.fullname template |
 | networkPolicy | object | `{"additionalAllowFroms":[],"enabled":true}` | Define a default NetworkPolicy for allowing apps in the same 'app.kubernetes.io/part-of' group to communicate with eachother. ref: https://kubernetes.io/docs/concepts/services-networking/network-policies/ |
+| nlb | object | `{"enabled":false,"healthcheck":{"healthyThresholdCount":2,"intervalSeconds":10,"protocol":"TCP","timeoutSeconds":5,"unhealthyThresholdCount":2},"scheme":"internet-facing","targetType":"ip"}` | Configure an NLB to route traffic direct to pods (for non HTTP traffic) |
```
Comment (nabadger) [inline charts/standard-application-stack/README.md:189]: I didn't think nlb supported healthchecks - can you confirm? 

I don't see any options on the web-ui for this either

Comment (nabadger) [reply charts/standard-application-stack/README.md:189]: Ah, they are supported, but I'm not sure http/https options are...

Comment (barrymars) [reply charts/standard-application-stack/README.md:189]: not tried it, but https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/guide/service/annotations/#healthcheck-path seems to suggest they are

File: charts/standard-application-stack/CHANGELOG.md (hunk 1 above)
Comment (barrymars) [reply charts/standard-application-stack/CHANGELOG.md]: tweaked

**Review by nabadger — APPROVED** [review]

**Pull request merged**